package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "start": "webpack-dev-server --mode development --config config/webpack.base.config.js --open --hot --history-api-fallback --env.PLATFORM=local --env.VERSION=stag",
     "prebuild": "webpack --mode production --config config/webpack.prod.config.js --env.PLATFORM=production --env.VERSION=stag --progress",
-    "build": "node server"
+    "build": "node server",
+    "lint": "eslintrc.json"
```

Comment:
You should use command to run linter, currently it's just config file name

Reply:
Fixed.